src/pages/ReportDetailsPage.js

```diff
@@ -93,15 +100,29 @@ function ReportDetailsPage(props) {
             return items;
         }
 
-        if (participants.length) {
+        if ((!isUserCreatedPolicyRoom && participants.length) || (isUserCreatedPolicyRoom && isPolicyMember)) {
```

Comment:
I'm not sure if we need the extra checks. Isn't this just the same as `participants.length`?